js/main.js

```diff
@@ -202,14 +210,50 @@ Site.Programacion = {
     bindToggle: function() {
       var _this = this;
 
-      $('.programacion-drawer-toggle').on('click', function() {
+      $('.programacion-title, .programacion-drawer-toggle, .close-drawer-overlay').bind('click', function() {
         _this.toggleContent();
       });
+
+      $('#programacion-slider').on({
+        mousewheel: function(event){
+          var delta = event.originalEvent.wheelDelta;
+
+          if(delta < 0 && !$('.programacion-drawer').hasClass('drawer-open')){
+            _this.toggleContent();
+          }
+        }
+      });
+
+
     },
 
     toggleContent: function() {
-      $('body').toggleClass('drawer-open');
+      var _this = this;
+
+      if ($('.programacion-drawer').hasClass('drawer-open')) {
+        $('.programacion-drawer').removeClass('drawer-open').css('height', '');
+        $('.programacion-content-holder').css('height', 'auto');
+        $('.close-drawer-overlay').hide();
+      } else {
+        _this.setDrawerHeight();
+        $('.programacion-drawer').addClass('drawer-open');
+        $('.close-drawer-overlay').show();
+      }
+
       $('.programacion-content-holder').scrollTop(0);
+    },
+
+    setDrawerHeight: function() {
+      var windowHeightHalf = $(window).height() / 2;
+      var drawerHeight = $('.programacion-header').outerHeight(true) + $('.programacion-content-holder').outerHeight(true);
+
+      if (drawerHeight > windowHeightHalf) {
```

Comment:
I'm not sure if this is working as expected for me. I am able to scroll the drawer in `.programacion-content-holder` but if it was full height it wouldn't make the drawer even half the height of the screen:

![screen shot 2016-09-29 at 12 57 38](https://cloud.githubusercontent.com/assets/845186/18965830/61d0183a-8644-11e6-8af4-bcec22428153.png)